<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serMainWindow</class>
 <widget class="QMainWindow" name="serMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2013</width>
    <height>963</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16666666</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(96, 195, 244);</string>
      </property>
      <property name="title">
       <string>SmartCare</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Poppins</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="general">
          <attribute name="title">
           <string>General</string>
          </attribute>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>9</y>
             <width>691</width>
             <height>671</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <family>Poppins</family>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(155, 232, 255);
font: 14pt &quot;Poppins&quot;;</string>
           </property>
           <property name="title">
            <string>service management</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout">
              <item row="7" column="0">
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="7" column="5">
               <widget class="QPushButton" name="pb_modifier">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 170, 255);</string>
                </property>
                <property name="text">
                 <string>Modify</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="le_ID">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item row="7" column="3">
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="6" column="2">
               <widget class="QLineEdit" name="le_qnt">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item row="8" column="2">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="2">
               <widget class="QLineEdit" name="le_name">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Dontype</string>
                </property>
               </widget>
              </item>
              <item row="7" column="6">
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Poppins</family>
                  <pointsize>14</pointsize>
                  <weight>50</weight>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>serviceName</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Donquantity</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Budget</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QComboBox" name="cb_type">
                <item>
                 <property name="text">
                  <string>food</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>blankets</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>clothes</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>money</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="2">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="2">
               <widget class="QLineEdit" name="le_budget">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="7" column="2">
               <widget class="QPushButton" name="pushButton_ajouter">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(85, 170, 127);</string>
                </property>
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="geometry">
            <rect>
             <x>730</x>
             <y>19</y>
             <width>991</width>
             <height>401</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(155, 232, 255);
font: 14pt &quot;Poppins&quot;;</string>
           </property>
           <property name="title">
            <string>show + PDF + excel + stats</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="0">
             <widget class="QTableView" name="tab_service">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>List of services :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="pb_pdf1">
                <property name="styleSheet">
                 <string notr="true">color: rgb(85, 170, 127);</string>
                </property>
                <property name="text">
                 <string>Generate PDF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="export_excel">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 170, 127);</string>
                </property>
                <property name="text">
                 <string>export excel</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="stats">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 170, 255);</string>
                </property>
                <property name="text">
                 <string>Stats</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="geometry">
            <rect>
             <x>730</x>
             <y>441</y>
             <width>1001</width>
             <height>391</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(155, 232, 255);
font: 14pt &quot;Poppins&quot;;</string>
           </property>
           <property name="title">
            <string>search + sort + PDF</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="le_recherche">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_recherche">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <item>
                 <property name="text">
                  <string>IDservice</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>serviceName</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dontype</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pb_recherche">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 170, 127);</string>
                </property>
                <property name="text">
                 <string>Search</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QTableView" name="recherche">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QComboBox" name="comboBox_sort">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <item>
                 <property name="text">
                  <string>IDservice</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>serviceName</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Donquantity</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pb_sort">
                <property name="styleSheet">
                 <string notr="true">color: rgb(85, 170, 127);</string>
                </property>
                <property name="text">
                 <string>sort</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pb_pdf2">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(255, 170, 255);</string>
                </property>
                <property name="text">
                 <string>Generate PDF</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="geometry">
            <rect>
             <x>9</x>
             <y>700</y>
             <width>691</width>
             <height>131</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(155, 232, 255);
font: 14pt &quot;Poppins&quot;;</string>
           </property>
           <property name="title">
            <string>delete service</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>IDservice</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_idsup">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_supprimer">
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 170, 127);</string>
                </property>
                <property name="text">
                 <string>delete</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <zorder>groupBox_8</zorder>
          <zorder>groupBox_2</zorder>
          <zorder>groupBox_6</zorder>
          <zorder>groupBox_9</zorder>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(133, 214, 255);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="logo"/>
   <addaction name="employees"/>
   <addaction name="destitute"/>
   <addaction name="donations"/>
   <addaction name="services"/>
   <addaction name="collects"/>
  </widget>
  <action name="collects">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Events.png</normaloff>:/images/Events.png</iconset>
   </property>
   <property name="text">
    <string>icon1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="employees">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Employees.png</normaloff>:/images/Employees.png</iconset>
   </property>
   <property name="text">
    <string>icon2</string>
   </property>
  </action>
  <action name="destitute">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Destitute.png</normaloff>:/images/Destitute.png</iconset>
   </property>
   <property name="text">
    <string>icon3</string>
   </property>
  </action>
  <action name="donations">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Donations.png</normaloff>:/images/Donations.png</iconset>
   </property>
   <property name="text">
    <string>icon4</string>
   </property>
  </action>
  <action name="services">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/Services.png</normaloff>:/images/Services.png</iconset>
   </property>
   <property name="text">
    <string>icon5</string>
   </property>
  </action>
  <action name="logo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/logo application.png</normaloff>:/images/logo application.png</iconset>
   </property>
   <property name="text">
    <string>logo</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
